lib/pbench/server/templates.py

```diff
@@ -25,17 +26,24 @@ class JsonFile:
     mapping file.
     """
 
-    def __init__(self, file: Path):
+    def __init__(self, file: Path = None):
         """
         Initialize a JSON file object by resolving the file name and reading
         the modification date.
 
         Args:
             file: JSON file path
         """
-        self.file = file.resolve(strict=True)
-        self.modified = datetime.fromtimestamp(self.file.stat().st_mtime)
-        self.json = None
+        if file:
+            self.file = file.resolve(strict=True)
+            self.modified = datetime.fromtimestamp(self.file.stat().st_mtime)
+            self.json = None
+
+    @staticmethod
+    def raw_json(json: Dict[AnyStr, Any]) -> "JsonFile":
+        new = JsonFile()
+        new.json = json
+        return new
```

Comment:
I see that, in the next commit, you're initializing them in the c'tor.

However, I'm still wondering if `raw_json()` should be setting the `modified` value (perhaps taking a parameter for the value, so that the caller can use `template.mtime`?).